packages/extension-polkagate/src/fullscreen/home/useProfileInfo.tsx

```
@@ -43,6 +44,19 @@ export default function useProfileInfo(label?: string | null): IconInfo {
           bgcolor: '#3988FF26',
           color: '#3988FF'
         };
+      case ADDRESS_BOOK_LABEL:
+        return {
+          Icon: Book,
+          bgcolor: '#74687626',
+          color: '#746876'
+        };
+
+      case ADDRESS_BOOK_LABEL:
```

Biome (2.4.4)

[error] Duplicate case label.

(lint/suspicious/noDuplicateCase)